sale_purchase_force_vendor/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+#. Go to *Sale -> Orders -> Quotations* and create a new Quotation.
+#. Create a new line with the following options:
+    * `Route`: Buy + MTO.
```

Comment:
```suggestion
    * `Route`: Buy
```

Reply:
It should be MTO for triggering the PO creation